app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -10,105 +10,81 @@
-          android:layout_marginBottom="32dp"
-          android:layout_marginEnd="8dp"
-          android:background="@drawable/reveal_hint_button_blue_background"
-          android:textSize="14sp"
-          android:text="@string/reveal_hint"
-          android:textColor="@color/white"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@+id/hint_title"
-          android:visibility="@{viewModel.isHintRevealed? View.GONE : View.VISIBLE}" />
+    <Button
+      android:id="@+id/reveal_hint_button"
+      style="@style/StateButtonActive"
+      android:layout_marginStart="8dp"
+      android:layout_marginTop="16dp"
+      android:layout_marginEnd="8dp"
+      android:layout_marginBottom="32dp"
+      android:background="@drawable/reveal_hint_button_blue_background"
+      android:minWidth="160dp"
+      android:minHeight="48dp"
+      android:text="@string/reveal_hint"
+      android:visibility="@{viewModel.isHintRevealed? View.GONE : View.VISIBLE}"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/hint_title" />
 
-        <FrameLayout
-          android:id="@+id/expand_list_icon"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="8dp"
-          android:layout_marginEnd="8dp"
-          android:layout_marginBottom="8dp"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintHorizontal_bias="0.966"
-          app:layout_constraintStart_toEndOf="@+id/hint_title"
-          app:layout_constraintTop_toTopOf="parent"
-          app:layout_constraintVertical_bias="0.0"
-          android:visibility="@{viewModel.isHintRevealed? View.VISIBLE : View.GONE}">
+    <ImageView
+      android:id="@+id/expand_list_icon"
+      android:layout_width="48dp"
+      android:layout_height="48dp"
+      android:layout_gravity="center_vertical"
+      android:layout_marginStart="8dp"
+      android:layout_marginBottom="8dp"
+      android:contentDescription="@{@string/show_hide_hint_list(viewModel.hintsAndSolutionSummary)}"
+      android:padding="8dp"
+      android:src="@drawable/ic_arrow_drop_down_black_24dp"
+      android:visibility="@{viewModel.isHintRevealed? View.VISIBLE : View.GONE}"
+      app:isRotationAnimationClockwise="@{isListExpanded}"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.82"
+      app:layout_constraintStart_toEndOf="@+id/hint_title"
+      app:layout_constraintTop_toTopOf="parent"
+      app:layout_constraintVertical_bias="0.0"
```

Comment:
Does this need to be defined if it's set to 0? Ditto for the other related layout files.

Reply:
Yes, since the default value is `0.5`.

Reply:
Ah right, that makes sense. Thanks.